ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse.yaml

```diff
@@ -37,8 +37,8 @@ tests:
 - as: installer-rehearse-azure
   interval: 960h
   steps:
-    cluster_profile: azure4
-    workflow: cucushift-installer-rehearse-azure-upi
+    cluster_profile: azuremag
```

Comment:
For the new cluster profile, I suggest to keep the original one, and create a new one for `azuremag` cluser_profile, that would be like this:
```
- as: installer-rehearse-azure
  interval: 960h
  steps:
    cluster_profile: azure4
    workflow: cucushift-installer-rehearse-azure-upi
- as: installer-rehearse-azure-mag
  interval: 960h
  steps:
    cluster_profile: azuremag
    workflow: cucushift-installer-rehearse-azure-ipi-azuremag
```
That will help other team member that not familiar with azure install to pick up a correct workflow for more integration testing based on the install workflow.